src/devices/bituo_technik.ts

```diff
@@ -18,10 +18,12 @@ const definitions: Definition[] = [
         configure: async (device, coordinatorEndpoint) => {
             const endpoint = device.getEndpoint(1);
             await reporting.bind(endpoint, coordinatorEndpoint, ['haElectricalMeasurement', 'seMetering']);
-            await reporting.currentSummDelivered(endpoint);
-            await reporting.currentSummReceived(endpoint);
+            await reporting.readMeteringMultiplierDivisor(endpoint);
```

Comment:
Why did you add both this line and `await endpoint.saveClusterAttributeKeyValue('seMetering', {divisor: 100, multiplier: 1});`, one overwrites the other.